poorwsgi/session.py

```diff
@@ -47,6 +51,43 @@ def hidden(text, passwd):
     return retval
 
 
+def get_token(secret, client, timeout=None, expired=0):
+    """Create token from secret, and client string.
+
+    If timeout is set, token contains time align with twice of this value.
+    Twice, because time of creating can be so near to computed time.
+    """
+    if timeout is None:
+        text = "%s%s" % (secret, client)
+    else:
+        if expired == 0:
+            now = int(time() / timeout) * timeout   # shift to start time
+            expired = now + 2 * timeout
+        text = "%s%s%s" % (secret, expired, client)
+
+    return sha256(text.encode()).hexdigest()
+
+
+def check_token(token, secret, client, timeout=None):
+    """Check token, if is right.
+
+    Arguments secret, client and expired must be same, when token was be
+    generated. If expired is set, than token must be younger than 2*expired.
```

Comment:
`was be` cannot be said in English. `Will be` (if you want future) or `was` (if you want past).